tests/e2e/plugins/index.js

```diff
@@ -25,11 +25,13 @@ module.exports = (on, config) => {
             })
         },
         deleteFolder(folderName) {
+            // eslint-disable-next-line no-console
             console.log('deleting folder %s', folderName)
 
             return new Promise((resolve, reject) => {
                 fs.rmdir(folderName, { maxRetries: 10, recursive: true }, (err) => {
                     if (err) {
+                        // eslint-disable-next-line no-console
```

Comment:
would it not be better to have the `// es-lint-disable no-console` at the top of the file instead ?

Reply:
Maybe for this one, but then I don't want any other console usage to be permitted anywhere else